<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Speed">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>vSpeed</string>
    </property>
   </widget>
   <widget class="QLabel" name="static_Speed_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sensitivity">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>vSensativity</string>
    </property>
   </widget>
   <widget class="QLabel" name="static_Sensativity_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensativity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Press to Select:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PressToSelect">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>vPressToSelect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Apply">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_Speed">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="placeholderText">
     <string>1-255</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_Sensativity">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="placeholderText">
     <string>1-255</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_Press_To_Select">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>0-1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
